UnityProject/Assets/IngameDebugConsole/Scripts/DebugLogUnitystationCommands.cs

```diff
@@ -440,5 +441,21 @@ private static void PlayStrandedEnding()
 			}
 		}
 
+
+		[ConsoleMethod("add-admin", "Promotes a user to admin using a user's account ID\nUsage: add-admin <account-id>")]
+		public static void AddAdmin(string userIDToPromote)
+		{
+			if (CustomNetworkManager.Instance._isServer == false)
+			{
+				Logger.LogError("Can only execute command from server.", Category.DebugConsole);
+				return;
+			}
+
+			var player = PlayerList.Instance.GetAdmin(ServerData.UserID, PlayerList.Instance.AdminToken);
+			if (player != null)
```

Comment:
No need to get admin here, if you can do server only console commands you should be consider an admin by default. Also this will prevent hosts from making admins if they haven't added themselves to the admin list yet

Reply:
Good point